tests/custom_managers/tests.py

```diff
@@ -512,6 +512,21 @@ def test_deconstruct_from_queryset_failing(self):
         with self.assertRaisesMessage(ValueError, msg):
             mgr.deconstruct()
 
+    def test_abstract_model_with_custom_manager_name(self):
+        """
+        Check that defined at the abstract class
```

Comment:
just state the expected behavior: "A custom manager may be defined on an abstract model. It will be inherited by the abstract model's children."